nova/virt/vmwareapi/cluster_util.py

```diff
@@ -131,7 +131,7 @@ def update_placement(session, cluster, vm_ref, group_info):
     """Updates cluster for vm placement using DRS"""
     cluster_config = session._call_method(
         vutil, "get_object_property", cluster, "configurationEx")
-
+    config_spec = None
```

Comment:
Why is this necessary?

Reply:
When I ran the tests one of them was failing with config_spec referenced before assignment.

Reply:
But this is not something that should happen in production, right? So shouldn't we fix the tests to look more like production instead?

Reply:
Actually we dont need it because we dont need a check there "cluster_config" will always has a value